sale_blanket_order/wizard/create_sale_orders.py

```diff
@@ -142,63 +137,53 @@ def create_sale_order(self):
         for customer in order_lines_by_customer:
             order_vals = {
                 'partner_id': customer,
-            }
-            if self.blanket_order_id:
-                order_vals.update({
-                    'origin': self.blanket_order_id.name,
-                })
-            order_vals.update({
-                'user_id': user_id if user_id else False,
-                'currency_id': currency_id if currency_id else False,
-                'pricelist_id': pricelist_id if pricelist_id else False,
-                'payment_term_id': (payment_term_id
-                                    if payment_term_id
-                                    else False),
+                'origin': self.blanket_order_id.name,
+                'user_id': user_id,
+                'currency_id': currency_id,
+                'pricelist_id': pricelist_id,
+                'payment_term_id': payment_term_id,
                 'order_line': order_lines_by_customer[customer],
-            })
+            }
             sale_order = self.env['sale.order'].create(order_vals)
             res.append(sale_order.id)
         return {
             'domain': [('id', 'in', res)],
-            'name': _('Sales Orders'),
```

Comment:
Why do you remove this? Shouldn't it be translatable?

Reply:
It already is: https://github.com/odoo/odoo/blob/12.0/odoo/addons/base/models/ir_actions.py#L173

Reply:
This is not an 'ir.actions.act_window' record, it is created on the go from python.
AFAIK, if you don't set the name to be translatable here, it will not appear in the pot file for its translation.

Reply:
done